web/public/js/controllers/manage-dojo-users-controller.js

```diff
@@ -172,6 +175,45 @@ function cdManageDojoUsersCtrl($scope, $state, $q, cdDojoService, alertService,
         }
       }
     },
+    sendEmail: {
+      ngClick: function () {
+        if ($scope.selectedItems.length !== 1) {
+          return cdDojoService.loadDojoUsers({dojoId: dojoId, userType: $scope.queryModel.userType, name: $scope.queryModel.name})
+          .then(function (users) {
+            var kids = _.filter(users.data.response, function (user) {
+              return _.includes(user.initUserType, 'attendee');
+            });
+            async.mapSeries(kids, function (kid, cb) {
+              cdUsersService.loadParentsForUserPromise(kid.id)
```

Comment:
Since this will take a long time, I propose allowing the user to type their message while we load this list.